AdminCore.Services/Configuration/DependencyInjection.cs

```diff
@@ -17,13 +18,14 @@ public static class DependencyInjection
   {
     private static bool _registered;
 
-    public static void RegisterDependencyInjection(IServiceCollection services = null)
+    public static void RegisterDependencyInjection(IServiceCollection serviceDescriptor = null,
```

Comment:
is the first parameter needed?

Reply:
The Startup.cs in WebApp uses IServiceCollection services in it's        DependencyInjection.RegisterDependencyInjection(services);
call, so I've included this to accept both